website/source/docs/guides/kubernetes-reference.html.md

```diff
@@ -0,0 +1,184 @@
+--- 
+layout: "docs"
+page_title: "Kubernetes Consul Reference Architecture"
+sidebar_current: "docs-guides-k8s-reference-architecture"
+description: |-
+  This document provides recommended practices and a reference architecture. 
+---
+
+# Consul and Kubernetes Reference Architecture
+
+Preparing your Kubernetes cluster to successfully deploy and run Consul is an
+important first step in your production deployment process. In this guide you
+will prepare your Kubernetes cluster, independent of its underlying platform
```

Comment:
nit: this sentence reads a little strangely to me. At first I thought it should be "independently" but that's not quite right either. Maybe just a rephrase:

> In this guide you will prepare a Kubernetes cluster than may be running on any platform (AKS, EKS, GKE, etc). However...

Note also - should be GKE to match the Kube-specific offering names for other clouds GCP ~= AWS ~= Azure while the other initialisms listed were their specific Kube offerings.

Reply:
Independently does stick out a bit here, I will update it.